site_scons/site_tools/extra/extra.py

```diff
@@ -21,20 +21,16 @@
 """SCons extra features"""
 from __future__ import print_function
 
+import subprocess
+import re
 import os
 
-# pylint: disable=no-name-in-module
-# pylint: disable=import-error
-from distutils.spawn import find_executable
 from SCons.Builder import Builder
 from SCons.Script import Dir
-# pylint: enable=no-name-in-module
-# pylint: enable=import-error
+from SCons.Script import WhereIs
```

Comment:
(pylint-no-name-in-module) No name 'Script' in module 'SCons'
(pylint-syntax-error) Cannot import 'SCons.Script' due to syntax error 'invalid syntax (<unknown>, line 58)'